scripts/build.sh

```diff
@@ -19,7 +19,7 @@ CURR_DIR=$( pwd )
 BUILD_DIR="${CURR_DIR}/rellic-build"
 INSTALL_DIR=/usr/local
 LLVM_VERSION=llvm-14
-CXX_COMMON_VERSION=v0.2.10
+CXX_COMMON_VERSION=v0.2.16
```

Comment:
This isn't strictly related, but I just bumped the version to get things building on my M1.